### ISSUE TITLE
[BUG] Square shape on MetaMask login screen when using transparencies

### ISSUE THREAD

**Issue by marcosdellavecchia** [family/connectkit#41]
**Describe the bug**

When setting the style property `--ck-body-background` to a color with transparency, the "MetaMask Login" screen displays a translucent square shape behind the content.

**To reproduce**

1. Apply the following custom style for the connect kit modal:
```
customTheme={{ '--ck-body-background': 'rgba(255, 255, 255, 0.04)' }}
```
2. Attempt to login using MetaMask.

**Expected behavior**

The screen displays a translucent square shape behind the modal content.

**Screenshots**

<img width="361" alt="Screen Shot 2022-11-09 at 20 59 09" src="https://user-images.githubusercontent.com/33458523/200968374-02c2c336-fb2b-468e-8cc1-597bafe57c5a.png">

**Environment details**

Google Chrome 107.0.5304.87

**Comment by lochie** [comment]
Good catch @marcosdellavecchia!
I can get this fixed in the next patch as this additional layer isn't necessary anymore (was needed to support an animation that has since been removed).

Will follow up when that's available ☺️

**Comment by lochie** [comment]
Hey @marcosdellavecchia, 

Thanks again for letting us know about this, this is now fixed in the latest version of ConnectKit (`1.0.1`). ✨
